sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,159 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+
+class CoulombKineticFriction(ActuatorBase):
+    """A friction force actuator for the Coulomb kinetic friction with Stribeck and viscous effect.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect, described by the following function:
+
+        f_f(v) = f_c * sign(v) + (f_max - f_c) * exp(-(v / v_s)^2) + sigma * v
+
+    where f_c is the Coulomb friction constant, f_max is the maximum static friction force,
+    v_s is the Stribeck velocity, sigma is the viscous friction constant, and v is the relative velocity.
+
+    Parameters
+    ==========
+
+    normal_force : Expr
+        The normal force between the surfaces.
+    mu_k : Expr
+        The coefficient of kinetic friction.
+    mu_s : Expr
+        The coefficient of static friction.
+    pathway : PathwayBase
+        The pathway that the actuator follows.
+    v_s : Expr
+        The coefficient of sliding friction (COF).
+    viscous_coeffient : Expr
+        The viscous friction coefficient.
+    """
+
+    def __init__(self, mu_k, mu_s, normal_force, pathway, *, v_s, viscous_coeffient=None):
```

Comment:
`mu_s` shouldn't be an argument, but a keyword argument if you need it, because the default kinetic coulomb friction doesn't depend on it.

Reply:
I changed to
```
def __init__(self, mu_k, normal_force, pathway, *, v_s, viscous_coeffient=None, mu_s=None):
        ...
        self._mu_s = mu_s if mu_s is not None else mu_k
```